cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/engine/sql/dataset/SQLDataset.java

```diff
@@ -14,14 +14,15 @@
  * the License.
  */
 
-package io.cdap.cdap.etl.engine;
+package io.cdap.cdap.etl.api.engine.sql.dataset;
+
+import io.cdap.cdap.api.data.schema.Schema;
 
 /**
- * Status for SQL Engine Jobs.
+ * Represents a dataset that resides in a SQL engine outside of spark.
  */
-public enum SQLEngineJobStatus {
-  CREATED,
-  RUNNING,
-  COMPLETED,
-  FAILED
+public interface SQLDataset {
+  String getDatasetName();
+  Schema getSchema();
+  long getNumRows();
```

Comment:
style: put newlines between the methods

Also, anything in an api module should have proper javadocs

Reply:
Added.

Reply:
Added javadocs.